travis/testDataHelpers.js

```diff
@@ -0,0 +1,108 @@
+const fs = require('fs');
+const process = require('process');
+const fhirInteractions = require('./fhirInteractions');
+
+/**
+ * Information about the test data available for a measure.
+ * 
+ * @typedef {Object} TestMeasureInfo
+ * @property {String} exmId - The EXM ID of the measure. ex. EXM_104
+ * @property {String} path - Path to the folder of test data.
+ * @property {String} measureReportPath - Path to the reference MeasureReport json file that should be compared against.
+ */
+
+/**
+ * Grabs the list of measures that fhir-patient-generator has test data for.
+ * 
+ * @returns {Promise<TestMeasureInfo[]>} List of information about each available test measure data.
+ */
+async function getTestMeasureList() {
+  // Find applicable measure test data directories in fhir-patient-generator
+  let fpgDir = fs.readdirSync('./fhir-patient-generator/')
+  let applicableMeasuresDirs = fpgDir.filter((dir) => { return dir.startsWith("EXM_") });
```

Comment:
I don't think it's that brittle since the measures are very distinct with the prefix. This code should weed out the macOS cruft as a side effect. But it's an easy thing to update, so go for it.

Reply:
Was just thinking we will definitely have other prefixes in the future besides `EXM_`, namely `CMS_`. I didn't consider macOS hidden files. Maybe the best approach would be to read the contents of the parent folder and then filter on folders. But yeah I'll go ahead an create the task.

Reply:
Yeah, definitely warrants further discussion and organization.